docs/Взаимодействие MS Word и MS Excel/Инструмент Рассылки.md

```diff
@@ -0,0 +1,55 @@
+# Инструмент рассылки
+Предположим, что у нас имеется вот такая таблица Excel:
+![Alt text](Инструмент-Рассылки.assets/image.png)
```

Comment:
[ch-md-error-image-alt-text](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-image-alt-text)